trading_calendars/exchange_calendar_xkar.py

```diff
@@ -327,6 +335,7 @@ def regular_holidays(self):
         '2016-12-12',
         '2017-12-01',
         '2018-11-21',
+        '2020-10-30',
```

Comment:
In the PR description you said 2020-10-29, is it actually 30?

Reply:
I typo'd in the comment, it should be the 30th per https://www.psx.com.pk/psx/files/?file=140396-1.pdf